.github/workflows/self-hosted.yml

```diff
@@ -0,0 +1,18 @@
+name: Add self-hosting issues to the self-hosting project
+
+on:
+  issues:
+    types:
+      - opened
+      - labeled
+
+jobs:
+  add-to-project:
+    name: Add issue to project
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/add-to-project@RELEASE_VERSION
```

Comment:
how does RELEASE_VERSION work here?

Reply:
You're right, this should have been `v1`. I just copy-pasted this from the docs without realising that it was placeholder text.